client/app/components/Footer/index.js

```diff
@@ -4,39 +4,20 @@ import Reveal from 'react-reveal';
 
 import styles from './styles.scss';
 
+import LargeLogo from './white.png';
```

Comment:
``` Javascript
ERROR in ./app/components/Footer/index.js
Module not found: Error: Can't resolve './white.png' in '/home/shawn/Documents/idicia/client/app/components/Footer'
 @ ./app/components/Footer/index.js 7:0-36
 @ ./app/containers/App/index.js
 @ ./app/app.js
 @ multi main
```

Looks like white.png was added to the folder `footer/` instead of `Footer/` so the relative link `./white.png` will fail to resolve, this should've caused an error when you tried doing yarn start/build...

This is irrelvant though, white.png already is hosted [here](http://res.cloudinary.com/dyjo5a3ci/image/upload/v1514087342/idicia/white.png) so we don't need to waste project space on images.